### PR TITLE
Fix occ user:add-app-password

### PR DESCRIPTION
**Pull request by nickvergessen**
### Steps
Create an app-password via all 3 methods and check the differences:

* occ user:add-app-password admin
* curl -k -u admin:admin https://nextcloud.local/ocs/v2.php/core/getapppassword -H "OCS-APIRequest: true"
* https://nextcloud.local/index.php/settings/user/security

### CODE REVIEW

File: core/Command/User/AddAppPassword.php [hunk 1]
```diff
@@ -108,11 +109,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 			return 1;
 		}
 
-		$output->writeln('<info>The password is not validated so what you provide is what gets recorded in the token</info>');
-
+		if (!$this->userManager->checkPassword($user->getUID(), $password)) {
+			$output->writeln('<error>The provided password is to be invalid</error>');
```
Comment (PVince81) [inline core/Command/User/AddAppPassword.php:113]: "is to be valid" or "is invalid" ?

**Review by PVince81 — APPROVED** [review]
:+1: please fix the typo

**Review by SMillerDev — COMMENTED** [review]
LGTM but then I thought the original implementation was fine too 😅

File: core/Command/User/AddAppPassword.php (hunk 1 above)
Comment (nickvergessen) [reply core/Command/User/AddAppPassword.php]: Fixed

**Review by CarlSchwan — APPROVED** [review]

**Pull request merged**